<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditTemperature">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFanSpeed">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>100</y>
     <width>113</width>
     <height>30</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonIncreaseTemp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>plus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDecreaseTemp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>minus</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFanLow">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>220</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Low</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFanMedium">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>220</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Med</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFanHigh">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>High</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Trenutna Temp</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>70</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Brzina</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelACStatus">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>68</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonToggleAC">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>360</y>
     <width>91</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>on/off</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
